release_notes/ose_3_1_release_notes.adoc

```diff
@@ -175,4 +194,11 @@ Features Support Scope]
 
 The following features are in Technology Preview:
 
-- Binary builds, and the `*Dockerfile*` source type for builds.
+* Binary builds, and the Dockerfile source type for builds.
+* Support for OpenShift Enterprise running on Atomic Host.
+* Containerized installations, meaning all OpenShift Enterprise components running in a container.
+
+== Known Issues
+
+* When pushing an internal registry when multiple registries share the same NFS volume, there is a chance the push will fail. A work-around has been suggested.
```

Comment:
"When pushing _to_ an internal registry"? Maybe?

s/work-around/workaround/ (per IBM Style Guide)

Where is the suggested workaround?